pkgs/development/tools/vcstool/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchurl, python35Packages }:
+
+  python35Packages.buildPythonApplication rec {
+    name = "vcstool";
+    version = "0.1.31";
+    #no tests
+    doCheck = false;
+
+    propagatedBuildInputs = with python35Packages; [ pyyaml ];
+    src = fetchurl {
+        url = "mirror://pypi/v/vcstool/${name}-${version}.tar.gz";
+        sha256 = "0n2zkvy2km9ky9lljf1mq5nqyqi5qqzfy2a6sgkjg2grvsk7abxc";
+    };
+
+    meta = with stdenv.lib; {
```

Comment:
A maintainer must be set, did you want to maintain this?

Reply:
sure, can I update the PR?

Reply:
Yep, I see that you've already done it. :+1: